project_forecast_line/models/project_task.py

```diff
@@ -116,18 +123,18 @@ def _update_forecast_lines(self):
                 continue
             date_start = max(today, task.forecast_date_planned_start)
             date_end = max(today, task.forecast_date_planned_end)
-            employee_ids = task.mapped("user_ids.employee_id").ids
-            if not employee_ids:
-                employee_ids = [False]
+            employees = task.mapped("user_ids.employee_id") or [False]
             _logger.debug(
                 "compute forecast for task %s: %s to %s %sh",
                 task,
                 date_start,
                 date_end,
                 task.remaining_hours,
             )
-            forecast_hours = task.remaining_hours / len(employee_ids)
-            for employee_id in employee_ids:
+            forecast_hours = task.remaining_hours / len(employees)
+            for employee_id in employees:
+                if self._update_only_active_employee() and not employee_id.active:
```

Comment:
If you end up keeping this. be careful there's a bug here.
On line 128, `employees` can be `[False]`, so you'd get attr error by trying to read `employee_id.active`